tests/brep/test_brep.py

```diff
@@ -0,0 +1,23 @@
+from pytest import fixture
+
+from compas.geometry import Box
+
+from compas_occ.brep import BRep
+
+
+@fixture
+def box():
+    return Box.from_width_height_depth(5., 5., 5.,)
+
+
+def test_create_brep():
+    _ = BRep()
+
+
+def test_brep_from_box(box):
+    _ = BRep.from_box(box)
```

Comment:
same here

Reply:
in this case we could also compare the volume of the created brep to the volume of the original box...